ui/app/rif/pages/lumino/explorer/explorer.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {pageNames} from '../../index';
+import rifActions from '../../../actions';
+import PropTypes from 'prop-types';
+
+class ExplorerPage extends Component {
```

Comment:
Why is this called explorer page? what is the purpose of this screen?

Reply:
It was just for "dev-ing", there i changed its name.